api/routes/user.js

```diff
@@ -1,6 +1,6 @@
 const express = require('express');
 const router = express.Router();
-const controllers = require('../controllers/user');
+const controllers = require('../controllers/auth');
 
 router.post('/login', controllers.login);
 
```

Comment:
Replace user.js on auth.js

Reply:
- is a deleted row, + - is an added row. So, I don't understand what you want to say.

Reply:
I mean file name

Reply:
done